index.bs

```diff
@@ -430,6 +432,13 @@ about current and past use of the API.
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="user-triggered-activation" dfn>User triggered activation</h4>
+
+The information whether the user agent is requested to access [=sensor readings=] due to
+[=triggered by user activation|user triggered activation=], may be used as a
+[=new information about the user's intent|criteria=] to grant access to [=sensor readings|readings=].
+The user agent may prompt user or apply other mitigation strategy when [=sensor readings=] are
+requested without user consent.
```

Comment:
let's not conflate user consent and user triggered activation, so I suggest: s/user consent/[=being triggered by user activation=]/